### PR TITLE
Remove OAS contract monitoring feature

### PR DESCRIPTION
**Pull request by RoyUP9**
Removes the feature added with https://github.com/up9inc/mizu/pull/325

Part of https://up9.atlassian.net/browse/TRA-4610

### CODE REVIEW

**Review by mertyildiran — DISMISSED** [review]

**Review by mertyildiran — APPROVED** [review]

File: tap/extensions/kafka/Makefile [hunk 1]
```diff
@@ -13,4 +13,4 @@ test-pull-bin:
 
 test-pull-expect:
 	@mkdir -p expect
-	@[ "${skipexpect}" ] && echo "Skipping downloading expected JSONs" || gsutil  -o 'GSUtil:parallel_process_count=5' -o 'GSUtil:parallel_thread_count=5'  -m cp -r gs://static.up9.io/mizu/test-pcap/expect9/kafka/\* expect
+	@[ "${skipexpect}" ] && echo "Skipping downloading expected JSONs" || gsutil  -o 'GSUtil:parallel_process_count=5' -o 'GSUtil:parallel_thread_count=5'  -m cp -r gs://static.up9.io/mizu/test-pcap/expect11/kafka/\* expect
```
Comment (nimrod-up9) [inline tap/extensions/kafka/Makefile:16]: Unrelated?

**Review by nimrod-up9 — APPROVED** [review]
BE is approved.
